adal/src/main/java/com/microsoft/aad/adal/HashMapExtensions.java

```diff
@@ -110,4 +113,36 @@ private HashMapExtensions() {
         return response;
     }
 
+    static HashMap<String, String> jsonStringAsMap(String bodyStr) throws JSONException {
```

Comment:
Since there's potentially some good reusability on this method, can you add a quick blurb of javadoc to summarize what it does, what it's inputs/outputs are?